code_challenges/adhoc/ad_hock_challenge.py

```diff
@@ -0,0 +1,32 @@
+#Write a function that takes a dictionary, string, and iterable of values.
+# The dictionaries keys will be strings and the values will be functions.
+# The return value should be the return value from the function from the input dict,  at the input string, 
+# when called with the values in the input iterable as arguments.
+def find_the_function(functions, function_name, the_numbers):
+    calling = functions[function_name]
+    return calling(the_numbers)
+    
+def adding(the_numbers):
```

Comment:
> values in the input iterable as arguments

This function is supposed to take the values IN the iterable as arguments, not the iterable of values.  So should look like:
`def adding(num1, num2, num3):`